course/templates/course/sign-in-email.txt

```diff
@@ -8,4 +8,4 @@ You have received this email because someone (maybe you) entered your email addr
 
 If this was not you, it is safe to disregard this email.
 {% endblocktrans %}
-{% blocktrans %}- RELATE staff {% endblocktrans %}
+- {{ RELATE }} {% trans "staff" %}
```

Comment:
This seems to be hardcoding the phrasing "XYZ staff", which may or may not work in some languages.

Reply:
Ok.